.github/workflows/build-preview.yml

```
@@ -4,6 +4,7 @@ on:
   pull_request:
     branches:
       - main
+  workflow_dispatch:
```

This CI script relies on `github.event.number`, which will be `undefined` if the script is triggered using `workflow_dispatch`. So I think we'll need to configure [an input](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputs):

```suggestion
  workflow_dispatch:
    inputs:
      pr_number:
        required: true
        type: number
```

This will also require changes to other parts of the script:
```diff
- echo ${{github.event.number}} > ./pr_number
+ echo ${{github.event.inputs.pr_number || github.event.number}} > ./pr_number
```

After that, I think we'll also have to change this line:

https://github.com/tordans/id-tagging-schema/blob/3cdd38dfad5026563382a2d000623bf38e7abc7f/.github/workflows/deploy-preview.yml#L12

I'm less familiar with this syntax so I don't have exact suggestion

---

If I see this correctly, we can just omit the `github.event.workflow_run.event == 'pull_request' &&` part in the `deploy-preview.yml` workflow, as it only runs when the `"Build Preview"` workflow is _completed_ (and that one is only triggered by pull requests and with this PR also by manual triggering it), which is precisely when we want the deploy step to happen.